website/docs/r/s3_bucket_policy.html.markdown

```diff
@@ -12,20 +12,37 @@ Attaches a policy to an S3 bucket resource.
 
 ## Example Usage
 
-### Using versioning
+### Creating a custom policy and adding to the bucket
```

Comment:
Sure.  I did a spot check on a couple of other pages and guessed that there needed to be some language here.  If it's not required, drop it.  I was more concerned with removing the "Using versioning" part.